lib/mayaUsd/fileio/orphanedNodesManager.cpp

```diff
@@ -207,22 +208,24 @@ void OrphanedNodesManager::handleOp(const Ufe::SceneCompositeNotification::Op& o
             }
             auto parentPrim = parentUsdItem->prim();
             bool foundChild { false };
-            for (const auto& c :
+            for (const auto& child :
                  parentPrim.GetFilteredChildren(UsdPrimIsDefined && !UsdPrimIsAbstract)) {
-                auto cPath = parentItem->path().popSegment();
-                cPath = cPath
+                auto childPath = parentItem->path().popSegment();
+                childPath
+                    = childPath
                     + Ufe::PathSegment(
-                            c.GetPath().GetAsString(), MayaUsd::ufe::getUsdRunTimeId(), '/');
+                          child.GetPath().GetAsString(), MayaUsd::ufe::getUsdRunTimeId(), '/');
 
-                auto ancestorNode = pulledPrims().node(cPath);
+                auto ancestorNode = pulledPrims().node(childPath);
                 // If there is no ancestor node in the trie, this means that
                 // the new hierarchy is completely different from the one when
                 // the pull occurred, which means that the pulled object must
                 // stay hidden.
-                if (ancestorNode) {
```

Comment:
Since the comment was describing the case where the if was not taken, I've inverted the condition so that the command now describes the if taken. Otherwise, it comment was confusing since it stated the opposite of the what was going on with the code.